packages/server/src/koaJsonRpc/index.ts

```diff
@@ -111,8 +113,8 @@ export default class KoaJsonRpc {
         return;
       }
 
+      ctx.state.methodName = body.method;
       const methodName = body.method;
-      const messagePrefix = `${requestIdPrefix} [POST] ${methodName}:`;
```

Comment:
Removing this line causes the requestIdPrefix on line 98 above to no longer be needed.  It can also be removed.

Reply:
Good catch. Removed